freyja/_cli.py

```diff
@@ -1,26 +1,30 @@
-import click
-import pandas as pd
-from freyja.convert_paths2barcodes import parse_tree_paths, \
-    convert_to_barcodes, reversion_checking, check_mutation_chain
-from freyja.read_analysis_tools import extract as _extract, \
-    filter as _filter, \
-    covariants as _covariants, \
-    plot_covariants as _plot_covariants
-from freyja.sample_deconv import buildLineageMap, build_mix_and_depth_arrays, \
-    reindex_dfs, map_to_constellation, solve_demixing_problem, \
-    perform_bootstrap
-from freyja.updates import download_tree, convert_tree, \
-    get_curated_lineage_data, get_cl_lineages, \
-    download_barcodes, convert_tree_custom
-from freyja.utils import agg, makePlot_simple, makePlot_time, \
-    make_dashboard, checkConfig, get_abundance, calc_rel_growth_rates, \
-    collapse_barcodes
-import os
```

Comment:
Any particular reason for the reorganization? Just a style choice?

Reply:
I ran isort on it, I think that is the reason it is changed